.devcontainer/socket_proxy.py

```
@@ -0,0 +1,90 @@
+    print(os.getpid(), flush=True)
+
+    server = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+    if os.path.exists(listen_path):
+        try:
+            os.remove(listen_path)
+        except Exception:
+            pass
+            
+    server.bind(listen_path)
+    # Allow read/write access to the socket for the container user
+    os.chmod(listen_path, 0o666)
```

Ruff (0.15.18)

[error] `os.chmod` setting a permissive mask `0o666` on file or directory

(S103)